app/views/employments/EndEmploymentView.scala.html

```diff
@@ -15,41 +15,89 @@
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcPageHeadingLegend
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcSectionCaption
+
+@this(mainTemplate: MainTempleteNew, form: FormWithCSRF,  govukDateInput : GovukDateInput, govukButton: GovukButton, govukErrorSummary: GovukErrorSummary)
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@(dateForm: Form[LocalDate], viewmodel: EmploymentViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser,  ec: ExecutionContext)
 
-@(dateForm: Form[LocalDate], viewmodel: EmploymentViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,  ec: ExecutionContext)
 
-@header = @{
-    ptaHeader(
-        formForErrorSummary = Some(dateForm),
-        displayBackLink = true,
-        mainHeadingText = Messages("tai.endEmployment.endDateForm.title", viewmodel.employerName),
-        preHeadingText = Messages("tai.endEmployment.preHeadingText")
-    )
+@hint = {
+    <div>
+        <span id="date-you-left-hint" class="govuk-!-font-weight-bold govuk-!-display-block" >@Messages("tai.endEmployment.endDateForm.label" , viewmodel.employerName)</span>
+        <span id="date-example-hint">@Messages("tai.label.date.example")</span>
+    </div>
 }
 
-@mainTemplate(title = Messages("tai.endEmployment.endDateForm.pagetitle"),
-      authedUser = Some(user),
-      pageHeader = Some(header),
-      articleClasses = None) {
+@mainTemplate(
+    title = Messages("tai.endEmployment.endDateForm.pagetitle"),
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    authedUser = Some(user),
+    backLinkId = "backLink",
+    backLinkContent = Some(Messages("tai.back-link.upper"))
+     ) {
 
     @form(action = controllers.employments.routes.EndEmploymentController.handleEndEmploymentPage(viewmodel.empId)) {
-        @date(
-            form = dateForm,
-            fieldPrefix = "tellUsAboutEmploymentForm",
-            labelText = Messages("tai.endEmployment.endDateForm.label", viewmodel.employerName),
-            legendBold = true
+
+
+        <div class = "govuk-grid-row">
+            <div class = "govuk-grid-column-two-thirds">
+
+        @if(dateForm.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummary(errorList = dateForm.errors.asTextErrorLinks, title = Text(messages("tai.income.error.form.summary"))))
+        }
+
+
+        @govukDateInput(DateInput(
+            id = "tellUsAboutEmploymentForm",
+            namePrefix = Some("tellUsAboutEmploymentForm"),
+            hint = Some(Hint(content = HtmlContent(hint))),
+            items = Seq(
+                InputItem(
+                    name= "day",
+                    label = Some(messages("tai.label.day")),
+                    classes = s"govuk-input--width-2 ${if(dateForm.hasErrors)  " govuk-input--error" else "" }",
```

Comment:
are classes needed? i think if you use errorMessage with a filter by key should highlight only the one that has an error

Reply:
discussed with Sergi - we can ignore this as it may not be possible with this component.

Reply:
yup here's an example: https://github.com/hmrc/manage-transit-movements-amendment-frontend/blob/19e8c66587ba6647cd763ecc9a61a90a6922083c/app/viewmodels/govuk/DateFluency.scala

InputItem doesn't allow an errorMessage so LGTM